src/pages/Course/Course.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  DefaultButton as Button,
+  DefaultCard as Card,
+  DefaultCardContent as CardContent,
+  DefaultSkeleton as Skeleton,
+  DefaultTypography as Typography,
+  DefaultBox as Box
+} from '@common/components'
+import { Stack } from '@mui/system'
+import { AuthContext } from '@services'
+import log from 'loglevel'
+import React, { useContext, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate, useParams } from 'react-router-dom'
+import { useLearningPathTopic } from '../../components/LocalNav/LocalNav.hooks'
+
+/**
+ * The Course component presents an overview of the course.
+ *
+ * @returns {JSX.Element} - The Course component.
+ *
+ * @category Pages
+ */
+export const Course = () => {
+  const { t } = useTranslation()
+  const authcontext = useContext(AuthContext)
+  const navigate = useNavigate()
+  const { courseId } = useParams() as { courseId: string }
+
+  const { loading, topics } = useLearningPathTopic(courseId)
+
+  useEffect(() => {
+    const preventEndlessLoading = setTimeout(() => {
+      log.log('Course timeout')
+      navigate('/login')
+    }, 5000)
+    if (authcontext.isAuth) {
+      clearTimeout(preventEndlessLoading)
+    }
+
+    return () => {
+      clearTimeout(preventEndlessLoading)
+    }
+  }, [authcontext.isAuth])
```

Comment:
glaube kann man entfernen, weil die Komponente eh neu rendert, sobald sich der context ändert.
```suggestion
  }, [])
```